libsolidity/ast/Types.cpp

```diff
@@ -3423,12 +3423,10 @@ MemberList::MemberMap FunctionType::nativeMembers(ASTNode const* _scope) const
 	}
 	case Kind::DelegateCall:
 	{
-		auto const* functionDefinition = dynamic_cast<FunctionDefinition const*>(m_declaration);
-		solAssert(functionDefinition, "");
-		solAssert(functionDefinition->visibility() != Visibility::Private, "");
-		if (functionDefinition->visibility() != Visibility::Internal)
+		if (auto const* functionDefinition = dynamic_cast<FunctionDefinition const*>(m_declaration))
 		{
-			auto const* contract = dynamic_cast<ContractDefinition const*>(m_declaration->scope());
+			solAssert(functionDefinition->visibility() > Visibility::Internal, "");
+			auto const *contract = dynamic_cast<ContractDefinition const*>(m_declaration->scope());
```

Comment:
Damn, I keep staring at this line and scratching my head wondering WTF the diff is broken :P

```suggestion
			auto const* contract = dynamic_cast<ContractDefinition const*>(m_declaration->scope());
```

Reply:
I git blamed `scripts/check_style.sh` and have determined that this is your fault. :smile: 

In any case, is it OK if I ignore this, since it's gonna get reformatted once I introduce clang-format at some point in the future? Because I've actually seen quite a few occurrences of bad style that is not caught by `check_style`, and don't feel like waiting for another build to finish for this.

Reply:
What, did `check_style.sh` actually detect this in the past?

In any case I approved the PR already, it was just so confusing to see this line again in the diff. Why doesn't github highlight this in the diff?

Reply:
> What, did check_style.sh actually detect this in the past?

Nope

Reply:
Well, `check_style.sh` is not my invention anyway. It's an ancient thing :P